Maple2.Server.Game/Manager/Items/ItemManager.cs

```diff
@@ -42,6 +49,45 @@ public class ItemManager {
         return item ?? Equips.Outfit.Values.FirstOrDefault(outfit => outfit.Uid == uid);
     }
 
+    public Item CreateItem(ItemMetadata itemMetadata, int rarity = 1, int amount = 1, bool initialize = true) {
+        var item = new Item(itemMetadata, rarity, amount, initialize);
+        item.Stats = itemStatsCalc.GetStats(item);
+        item.Socket = itemStatsCalc.GetSockets(item);
+        
+        
+        item.Appearance = itemMetadata.SlotNames.FirstOrDefault(EquipSlot.Unknown) switch {
```

Comment:
Looks like `item.Appearance` is mutable, so you can just do:
```cs
item.Appearance.Color = GetColor(item);
```

The appropriate Appearance type is already initialized in the constructor.